### PR TITLE
dracula theme

### PR DESCRIPTION
**Pull request by comforttiger**

### CODE REVIEW

**Comment by wavebeem** [comment]
I'm not sure how easy it is to target the same tokens as VSCode since this uses a completely different syntax highlighting library, but I wanted to post a comparison since I noticed a few differences:

### Dracula Official for VSCode

image

### Dracula from this PR

image

File: src/themes/dracula.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    margin: "1.25rem -0.75rem",
+    "border-radius": "0",
+    "line-height": "1.5",
+    "overflow-x": "auto",
+    background: ui.bg0,
+    color: syntax.default,
+    "color-scheme": "dark",
+    "scrollbar-color": `${ui.bg1} ${ui.fg}`,
+    "border-top": "1px solid #ccc",
+    "border-bottom": "1px solid #ccc",
```
Comment (wavebeem) [inline src/themes/dracula.ts]: These borders look like they were copied from the classic light theme. They don't look good on a dark theme to me. The rest of my dark themes don't have borders, so I think you can safely remove them.

File: src/themes/dracula.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  _footer: {
+    margin: "-0.75rem",
+    marginTop: "-1.25rem",
+    marginBottom: "1.25rem",
+    padding: "0.25rem 0.5rem",
+    fontSize: "smaller",
+    textAlign: "right",
+    background: ui.bg1,
+    color: ui.fg,
+    "border-bottom": "1px solid #ccc",
```
Comment (wavebeem) [inline src/themes/dracula.ts]: No border needed here

File: src/themes/dracula.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  },
+
+  _footer: {
+    margin: "-0.75rem",
+    marginTop: "-1.25rem",
+    marginBottom: "1.25rem",
+    padding: "0.25rem 0.5rem",
+    fontSize: "smaller",
+    textAlign: "right",
+    background: ui.bg1,
```
Comment (wavebeem) [inline src/themes/dracula.ts]: This should probably be ui.bg0

**Comment by comforttiger** [comment]
i don't think that this theme will be able to match 100% up to what it looks like in vscode, but i can try to see if i can do something about some of the differences

**Comment by wavebeem** [comment]
Yeah, no worries at all about it not looking 100% identical. That's one of the reasons I haven't tried to add any 3rd party themes. Just leave a comment when you'd like me to look at this again!

**Comment by comforttiger** [comment]
i think this is the closest i can get with the tokens prismjs has

**Comment by wavebeem** [comment]
I'm going to merge this into another branch and see if I can make some tweaks to the code that actually applies the theme rules. I'm noticing some bugs right now and I'd like to fix them before getting this in.

**Pull request merged**